### DISCUSSION TITLE
Some Privacy Queries

### DISCUSSION THREAD

**Discussion by FrozenVertx** [anonaddy/anonaddy#228] [General]
Hi,
I recently read privacy policy as I use service everyday and I want  to ask of 2 questions regarding to privacy policy that i don't understand or not mention in it.

1. As we know Anonaddy not store email on server then is it possible to start store or log emails elsewhere after governments orders to handover data and spy on user emails afterwards without user consent?

2. Is anything planned or any update about #163 (_client-side E2E about user fields_) ?

**Comment by willbrowningme** [comment]
1. I believe this is covered by this section in the [privacy policy](https://anonaddy.com/privacy/):

> Server logs are only used to improve our service and prevent abuse or prohibited use. This information will not be provided under any circumstances to any parties other than when compelled by law.

2. No updates yet, if you are concerned about the description field not being end-to-end encrypted then I would advise you to leave it blank and store it in your password manager instead.

**Reply by FrozenVertx** [reply]
Hmm, So government access from server logs like IP,identity of any person. I still don't understand If gov come and say we want this person XYZ emails. So you say we have only server log (IP and other log of session). After that they order that we want this user XYZ mails so if any come mail comes in future then you have to forward to our gov email. And after that every email copy is forwarded to government.

 I asked above because 
1.[Recent Protonmail Reveal News](https://techcrunch.com/2021/09/06/protonmail-logged-ip-address-of-french-activist-after-order-by-swiss-authorities/)
2.if I think right when anonaddy receives email its unencrypted if pgp encryption applied between anonaddy and user not applied between sender and anonaddy. so from this concept its possible to forward whole email to government.
3. Concerned about sending any confidental email in future.
